service/src/main/java/bio/terra/workspace/service/workspace/flight/delete/cloudcontext/DeleteResourcesStep.java

```diff
@@ -52,12 +54,18 @@ public DeleteResourcesStep(
   //  @SuppressWarnings("unchecked")
   public StepResult doStep(FlightContext context) throws InterruptedException, RetryException {
     // Jackson does not deserialize the list properly with a TypeReference. The array form works.
-    ResourceDeleteFlightPair[] resourcePairs =
+    ResourceDeleteFlightPair[] resourcePairsArray =
         FlightUtils.getRequired(
             context.getWorkingMap(),
             RESOURCE_DELETE_FLIGHT_PAIR_LIST,
             ResourceDeleteFlightPair[].class);
 
+    List<ResourceDeleteFlightPair> resourcePairs = Arrays.asList(resourcePairsArray);
+    logger.info(
+        "Flight {} DeleteResourcesStep found {} resources to delete",
```

Comment:
would it be helpful to list all the type and id of the resources like you did in samservice?

Reply:
I'd have to fetch the resources from the database. That felt like more than I wanted to do for diagnostics.